decompiler/level_extractor/extract_level.cpp

```diff
@@ -323,8 +323,7 @@ void extract_common(const ObjectFileDB& db,
 
   if (config.rip_levels) {
     auto file_path = file_util::get_jak_project_dir() / "glb_out" /
-                     game_version_names[config.game_version] / "common.glb";
-    file_util::create_dir_if_needed_for_file(file_path);
+                     game_version_names[config.game_version] / "common";
     save_level_foreground_as_gltf(tfrag_level, art_group_data, file_path);
```

Comment:
dont get rid of the `create_dir_if_needed...` call here.  That is what prevents the issues of things not working if the directory doesn't already exist -- unless this was moved elsewhere (i dont see it)

Reply:
I just moved the create_dir_if_needed call into `save_level_foreground_as_gltf` right before we actually write the file. Since it now writes to multiple different filenames, I only have the prefix at this point - and it didn't seem to work properly if there wasn't a filename after the deepest folder in the path. For example, using ".../jak2/common" as the path would only create up to the jak2 folder and not the common folder, and then it would fail to write the common glbs because the folder wouldn't exist.